tests/client-sdk/inference/test_inference.py

```diff
@@ -371,6 +378,33 @@ def test_image_chat_completion_streaming(llama_stack_client, vision_model_id):
     assert any(expected in streamed_content for expected in {"dog", "puppy", "pup"})
 
 
+def test_image_chat_completion_base64_data(
+    llama_stack_client, vision_model_id, base64_image_data
+):
+    message = {
+        "role": "user",
+        "content": [
+            {
+                "type": "image",
+                "image": {
+                    "data": base64_image_data,
```

Comment:
I think this should be a binary byte representation of the image in "data" instead of base64 string?

Reply:
idk. the spec says base64 (see below). image.data is bytes though (see https://github.com/meta-llama/llama-stack/blob/main/llama_stack/apis/common/content_types.py#L24).

https://llama-stack.readthedocs.io/en/latest/references/api_reference/index.html#tag/Inference/paths/~1v1~1inference~1chat-completion/post

![image](https://github.com/user-attachments/assets/642e0046-2689-4950-b584-61f853f64c67)

Reply:
Thanks for pointing this out. Your original base64 obeying the spec is correct, the mismatch in our implementation which is fixed in: https://github.com/meta-llama/llama-stack/pull/909